airflow/providers/opensearch/hooks/opensearch.py

```diff
@@ -40,13 +44,20 @@ class OpenSearchHook(BaseHook):
     conn_type = "opensearch"
     hook_name = "OpenSearch Hook"
 
-    def __init__(self, open_search_conn_id: str, log_query: bool, **kwargs: Any):
+    def __init__(
+        self,
+        open_search_conn_id: str,
+        log_query: bool,
+        open_search_conn_class: type[OpenSearchConnectionClass] | None = RequestsHttpConnection,
```

Comment:
Is it required to change default OpenSearch Connection Type? I guess by default it use `urllib3` (None in this case)

Reply:
The Opensearch client indeed uses `urllib3` as default, but the hook actually defaults to `RequestsHttpConnection`. I chose to keep the hook's default to make sure nothing should break.